recipes/phidl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "phidl" %}
+{% set version = "1.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/phidl-{{ version }}.tar.gz
+  sha256: b2b1b9bd6d92fac6fc1028a69d410e220e712785e13ef6286a162c2a8976881f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >3.6
+  run:
+    - gdspy >=1.5
+    - matplotlib-base
+    - numpy
+    - python
```

Comment:
The same restriction on the python version under `host` should also be used under `run`.